app/src/main/java/org/fossasia/openevent/general/welcome/WelcomeFragment.kt

```diff
@@ -28,9 +27,12 @@ const val LOCATION_PERMISSION_REQUEST = 1000
 const val WELCOME_FRAGMENT = "welcomeFragment"
 
 class WelcomeFragment : Fragment() {
+    
     private lateinit var rootView: View
     private val geoLocationViewModel by viewModel<GeoLocationViewModel>()
     val preference = Preference()
+    private lateinit var editor: SharedPreferences.Editor
```

Comment:
Where is this used?

Reply:
Do not resolve conversations manually. It is automatically resolved once requested changes are made